app/utils/jwt.py

```diff
@@ -43,7 +43,7 @@ def get_admin_payload(token: str) -> Union[dict, None]:
 
 
 def create_subscription_token(username: str) -> str:
-    data = username + ',' + str(int(datetime.now().timestamp()))
+    data = username + ',' + str(round(datetime.utcnow().timestamp()))
```

Comment:
using datetime ( instead of timestamp ) its not better ?

Reply:
Timestamp is short and JWT used the same thing, so i just did the same